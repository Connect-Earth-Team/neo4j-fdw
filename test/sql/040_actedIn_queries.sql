\a
SELECT * FROM actedIn ORDER BY id DESC LIMIT 10;
SELECT * FROM actedIn WHERE movie_id = 0;
SELECT * FROM actedInWithCustomWhere1 WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';
SELECT * FROM actedInWithCustomWhere2 WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';
SELECT * FROM actedInWithCustomWheres WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';