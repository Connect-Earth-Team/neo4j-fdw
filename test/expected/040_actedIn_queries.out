\a
SELECT * FROM actedIn ORDER BY id DESC LIMIT 10;
id|movie_id|person_id
239|162|146
238|162|163
237|162|77
236|162|165
235|162|164
234|162|71
232|161|71
230|159|148
229|159|160
228|159|71
(10 rows)
SELECT * FROM actedIn WHERE movie_id = 0;
id|movie_id|person_id
7|0|8
3|0|4
2|0|3
1|0|2
0|0|1
(5 rows)
SELECT * FROM actedInWithCustomWhere1 WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';
actor|movie
Keanu Reeves|The Matrix
(1 row)
SELECT * FROM actedInWithCustomWhere2 WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';
actor|movie
Keanu Reeves|The Matrix
(1 row)
SELECT * FROM actedInWithCustomWheres WHERE movie = 'The Matrix' AND actor = 'Keanu Reeves';
actor|movie
Keanu Reeves|The Matrix
(1 row)
